packages/bezier-react/src/components/ProgressBar/ProgressBar.stories.tsx

```diff
@@ -160,10 +176,18 @@ export const UsageWidth: Story<{}> = () => (
     </StackItem>
   </VStack>
 )
-
 UsageWidth.storyName = 'Usage (width)'
+UsageWidth.parameters = {
```

Comment:
```suggestion
UsageWidth.parameters = docsParamters
```

다른 스토리도 마찬가지 인데, docs 모드의 파라미터를 상단에 선언해두고 같이 쓰면 좋겠습니다.
나중에 이 작성방식이 가이드로 올라갔을때 모드별 파라미터를 한 군데서 관리하면 전체적으로 통일되고 좋을 것 같습니다.

Reply:
템플릿이 확실히 정립되면 공통 모듈에 몰아넣어도 괜찮을 거 같습니다.